packages/components/src/range-control/README.md

```diff
@@ -40,12 +40,12 @@ Possible selections may be organized through the use of tick marks, which a thum
 
 ### Behavior
 
-- **Click and drag**: The slider is controlled by clicking the thumb and dragging it.
-- **Click jump**: The slider is controlled by clicking the track.
-- **Click and arrow**: The slider is controlled by clicking the thumb, then using arrow keys to move it.
-- **Tab and arrow**: The slider is controlled by using the tab key to select the thumb of the desired slider, then using arrow keys to move it.
-- **Value entry field**: Discrete sliders have value entry fields. After a text entry is made, the slider position automatically updates to reflect the new value.
-- **Tick marks** (Optional) Discrete sliders can use evenly spaced tick marks along the slider track, and the thumb will snap to them. Each tick mark should change the setting in increments that are discernible to the user.
+-   **Click and drag**: The slider is controlled by clicking the thumb and dragging it.
```

Comment:
A bunch of auto-formatting

Reply:
is this an editor config? I don't mind as long as we're consistent within the file (ideally, also within the repo). I'd perhaps look into why these are added? it seems weird to have 3 spaces instead of 1 in between the `-` and the text.

Reply:
@nosolosw I'm not sure? My VSCode auto formatted it like that. I have both Editorconfig and Prettier plugins installed.